cmd/fleetctl/user.go

```diff
@@ -174,6 +179,123 @@ func createUserCommand() *cli.Command {
 	}
 }
 
+func createBulkUsersCommand() *cli.Command {
+	return &cli.Command{
+		Name:  "create-users",
+		Usage: "Create bulk users",
+		UsageText: `This command will create a set of users in Fleet by importing a CSV file. Expected columns are: Name,Email,SSO,API Only,Global Role,Teams. Created Users by default get random password and Observer Role.
+		Password could be left blank, Users should receive an invitation link.`,
```

Comment:
While we are at it, I believe this doesn't apply anymore and should be removed: `Password could be left blank`.